### PR TITLE
fix description capitalization/wording

### PR DESCRIPTION
**Pull request by aditisundar**
- make violation description sentence-case
- fix hometown & vehicle descriptions for consistency

### CODE REVIEW

File: 911.js [hunk 1]
```diff
@@ -148,15 +148,19 @@ function grabData() {
     getTrafficData(src);
 }
 
+function toSentenceCase(s) {
+    return s[0] + s.slice(1).toLowerCase()
+}
```
Comment (AstroCB) [inline 911.js]: Nit: semicolon?

**Review by AstroCB — CHANGES_REQUESTED** [review]

File: 911.js (hunk 1 above)
Comment (kuang) [reply 911.js]: +1

Comment (aditisundar) [reply 911.js]: no

File: 911.js [hunk 2]
```diff
@@ -152,11 +152,11 @@ function getTrafficData(src) {
     $.getJSON(src, function (data) {
         data.forEach(dp => {
             let description = "<p>"
-                + dp.description
-                + "</br></br> This driver is from: "
-                + dp.driver_city + " "
+                + dp.description[0] + dp.description.slice(1).toLowerCase()
```
Comment (AstroCB) [inline 911.js:155]: I liked the function better – this is a little messy, but you can leave that for a follow-up.

**Review by AstroCB — APPROVED** [review]

File: 911.js (hunk 2 above)
Comment (kuang) [reply 911.js:155]: #remindMe in 4 years to update

**Pull request merged**